apps/client/src/common/components/view-params-editor/ParamInput.tsx

```diff
@@ -29,19 +34,19 @@ export default function ParamInput({ paramField }: EditFormInputProps) {
   }
 
   if (type === 'boolean') {
-    const defaultCheckedValue = isStringBoolean(searchParams.get(id)) ?? false;
+    const defaultCheckedValue = isStringBoolean(searchParams.get(id)) ?? defaultValue;
```

Comment:
`isStringBoolean` returns a boolean, so the nullish coalescing thing will never do its thing.  This likely not what you meant to write

This happens a few more times down the code